sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -942,21 +945,21 @@ def get_manifest(self, repository: str, tag_or_digest: str, **kwargs) -> GetMani
                 **kwargs
             )
         )
+        if response.http_response.headers['Content-Length'] > MAX_MANIFEST_SIZE:
+            raise ValueError("Manifest size is bigger than 4MB.")
```

Comment:
nit: Not sure if we standardized the error message across languages already, but I wonder if we should rephrase this to "Manifest size must be smaller than 4MB" so it's clear that >4MB is too big

Reply:
I think it should be fine, it's consistent with other languages.